shared/src/iosMain/kotlin/com/kcc/kmmhackathon/shared/utility/DateKMM.kt

```diff
@@ -0,0 +1,32 @@
+package com.kcc.kmmhackathon.shared.utility
+
+import platform.Foundation.*
+
+actual class DateKMM {
+
+    private var date: NSDate
+
+    actual constructor() {
+        date = NSDate()
+    }
+
+    actual constructor(time: Long) {
+        date = NSDate.dateWithTimeIntervalSince1970(time.toDouble())
+    }
+
+    constructor(platformDate: NSDate) {
+        date = platformDate
+    }
+
+    actual fun compareTo(other: DateKMM): Int {
```

Comment:
Is there any way we can add tests for this logic?

Reply:
Yeah currently doing it 👍

Reply:
Done